recipe/meta.yaml

```diff
@@ -24,15 +25,14 @@ requirements:
   host:
     - python
     - numpy
-    - pybind11 >=2.8.0
+    - pybind11 >=2.12.0
     - setuptools_scm >=7.0.0
     # explicit requirement and also required by setuptools_scm[toml]:
     - setuptools >=42
     - pip
   run:
     - python
-    - numpy <2.0.0a0
-    - scipy >=1.8.0
+    - scipy >=1.11.0
```

Comment:
Note: This comes from [this `requirements.txt` file]( https://github.com/pyamg/pyamg/blob/fb61aeb53d80a5fb59d332944e276e1363225c9a/requirements.txt#L2 )